PWGHF/TableProducer/candidateSelectorXicToPKPi.cxx

```diff
@@ -199,8 +245,13 @@ struct HfCandidateSelectorXicToPKPi {
 
       if (!topolXicToPKPi && !topolXicToPiKP) {
         hfSelXicToPKPiCandidate(statusXicToPKPi, statusXicToPiKP);
+        if (applyMl) {
+          hfMlXicToPKPiCandidate(outputMlXicToPKPi, outputMlXicToPiKP);
+        }
         continue;
       }
+      SETBIT(statusXicToPKPi, aod::SelectionStep::RecoTopol);
+      SETBIT(statusXicToPiKP, aod::SelectionStep::RecoTopol);
```

Comment:
Here you should set the flags accordingly to which mass hypothesis passed the selections.